### DISCUSSION TITLE
Redshift Connector?

### DISCUSSION THREAD

**Discussion by yhkdev** [sinaptik-ai/pandas-ai#1280] [Ideas]
Are there any plans on making PandasAI easy to use with AWS Redshift?

I am able to make the connection to Redshift using SQLConnector(), but it seems I have to initialize a separate SQLConnector() for each schema.table I want to chat with, then put them in a SmartDataLake(). It'd be great if we can just specify the schema in a single SQLConnector() and chat with all the tables in it.

**Comment by jbold569** [comment]
Do you mind sharing the connector parameters you used?
